conda.bzl

```diff
@@ -49,14 +64,16 @@ def _download_conda(rctx):
     os = get_os(rctx)
     arch = get_arch(rctx)
     ext = INSTALLER_SCRIPT_EXT_MAP[os]
-    url = CONDA_BASE_URL + CONDA_INSTALLER_NAME_TEMPLATE.format(major = CONDA_MAJOR, minor = CONDA_MINOR, os = os, arch = arch, ext = ext)
+
+    #url = CONDA_BASE_URL + CONDA_INSTALLER_NAME_TEMPLATE.format(major = CONDA_MAJOR, minor = CONDA_MINOR, os = os, arch = arch, ext = ext)
```

Comment:
I don't want to overengineer this. Maybe the easiest way is to pass an `attr` to `load_conda_rule = repository_rule(...` with `installer="miniconda"` as the default. You can pass `"miniforge"` as an alternative. We can add `mambaforge` or whatever fancy new thing later.

`_load_conda_impl` or `_download_conda` will check and use the right string as appropriate.

Overengineering would be to use a struct / dataclass / whatnot to hold the collection of strings.

Reply:
Definitely passing `installer` as a parameter with `miniconda` as the default value is a good idea. 

Using structs doesn't really feel like overengineering to me, might even make the code cleaner. But the other way is not bad also. So just follow your intuition 😛

Reply:
Done - with just `if / else`.